manifests/init.pp

```diff
@@ -115,6 +120,7 @@
   Stdlib::Fqdn $servername = $facts['networking']['fqdn'],
   Array[Stdlib::Absolutepath] $allowed_import_path = ['/var/lib/pulp/sync_imports'],
   Optional[String] $remote_user_environ_name = undef,
+  Integer[1, 24] $worker_count = min(8, $facts['processors']['count']), # lint:ignore:parameter_documentation
```

Comment:
Why do you ignore this? It is documented now.

Reply:
I was getting a lint failure that `$facts['processors']['count']` is not documented. (Interesting to me that we didn't see the same issue with `$facts['networking']['fqdn']` three lines above)

Reply:
https://github.com/voxpupuli/puppet-lint-param-docs/pull/12 should address that.